float64 lat
float64 lon
float64 alt
int32 gpsNum
int32 gpsStatue 
float32 gpsHdop
